src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/CmdletBase/ResourceWithParameterCmdletBase.cs

```diff
@@ -134,6 +134,7 @@ public object GetDynamicParameters()
                         MyInvocation.MyCommand.Parameters.Keys.ToArray());
                 }
             }
+            this.CopiedDynamicParameters = dynamicParameters;
```

Comment:
Hmm, this isn't the interface I was hoping for.  Is this the only place we can get DynamicParameters from?  

If we need to do this, I think a void method with a bit less esoteric name, like ```void RegisterDynamicParameters(RuntimeParamaterDictionary paremeters) might be a  more intuitive option for new usages.

Reply:
Unfortunately I could not find any way to get the dynamic parameters outside of copying them over during runtime.  I created the void method and updated the relevant files.